### ISSUE TITLE
Improve accessibility for outlines

### ISSUE THREAD

**Issue by meganvilla** [OpenSpace/OpenSpace#3105]
Missing Focus Indicator on Many Controls 
Severity: 1 

While some buttons show the default focus outline when reached by pressing Tab or  Shift+Tab, most interactive elements have the styling of "outline: 0" applied to remove  the focus outline. The lack of a focus indicator along with the inaccessible  implementation of many controls makes the page unusable for keyboard users as it is  impossible to realize which element is currently focused. 
Examples include: 

- The three vertical dots menu button; 
- The "Close" button in the "About OpenSpace" dialog; 
- The "Scene" control;
- The "Search" input field in the "Scene" menu pane; note that the field relies only on  its cursor as an indication that the field is reached, but this may be difficult to  notice; 
- The "Scene" custom menu items which expand or collapse additional options; and
- Color Picker buttons, sliders (See: Problematic Focus Indicator for Form Controls),  dropdowns, etc. 

**Suggested Solution** 
Remove the styling of "outline: 0" and provide a highly visible focus indicator for all  interactive controls. 

For details about requirements and best practices, see the following resources:
- https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html 
- https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance.html
- https://www.w3.org/WAI/WCAG22/Understanding/focus-not-obscured minimum.html
- https://sarasoueidan.com/blog/focus-indicators/

**Comment by alexanderbock** [comment]
Outlines achieved in e3f5b83f7476c8507dccd45ab98e4731504af532
